iOS_SDK/OneSignalSDK/UnitTests/Shadows/OneSignalClientOverrider.h

```diff
@@ -48,5 +48,7 @@
 +(void)disableExecuteRequestOverride:(BOOL)disable;
 +(NSArray<OneSignalRequest *> *)executedRequests;
 + (void)setMockResponseForRequest:(NSString *)request withResponse:(NSDictionary *)response;
++ (NSDictionary*)iosParamsResponse;
```

Comment:
We are using iOS SDK so I think we can remove ios and just call this 
`remoteParamsResponse`?
`remoteParams`?

Reply:
remoteParamsResponse 👍